Wiki/index.md

```diff
@@ -114,128 +110,100 @@ scientific computations instead of securing the blockchain.
 
 ## Official Resources
 
-  - Website: <https://gridcoin.us/>
-  - [Volunteers, Roles &
-    Privileges](Volunteers,-Roles,-&-Privileges "wikilink")
```

Comment:
Maybe comment this out with and add a TODO instead so we don't forget to add this page back in when eventually written.